hubspot/errors.go

```diff
@@ -50,7 +51,7 @@ type ErrLinks struct {
 }
 
 func (c *Connector) interpretError(res *http.Response, body []byte) error {
-	if res.Header.Get("Content-Type") == "application/json" {
+	if res.Header.Get("Content-Type") == "application/json;charset=utf-8" {
```

Comment:
If you're having to deal with this level of differentiation, I'd suggest using [mime.ParseMediaType](https://pkg.go.dev/mime#ParseMediaType). This will give you `application/json` for the first return value (even if the charset is set). It would behave properly regardless of whether or not the charset is added on. More robust, in other words.

Reply:
This is great, thank you!